data/schema/schema_v1.json

```diff
@@ -296,12 +296,19 @@
         },
         "sections": {
           "type": "array",
-          "description": "Title - initial title, title_from_answers - title with information derived from answers",
           "items": {
-            "title": "string",
-            "title_from_answers": "array",
-            "group_order": "array",
-            "description": "Order in which groups will appear during survey",
+            "title": {
+            "type": "string",
+            "description": "Initial title."
```

Comment:
- The formatting is slightly off here, `type` and `description` should be indented
- The description is a bit vague, if we can't think of anything of value it might be worth just dropping the description.

Reply:
Indentation done, title description amended to 

'Title for given id'